client/src/components/PublicPages/AuditPlanner/ElectionResultsCard.tsx

```diff
@@ -0,0 +1,295 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Button, Card, Colors, H2, InputGroup } from '@blueprintjs/core'
+
+import IntegerInput from './IntegerInput'
+import { Confirm, useConfirm } from '../../Atoms/Confirm'
+import {
+  constructInitialElectionResults,
+  constructNewCandidate,
+  ICandidate,
+  IElectionResults,
+} from './electionResults'
+import { StyledTable } from '../../Atoms/Table'
+
+const Container = styled(Card)`
+  padding: 0;
+`
+
+const InnerContainer = styled.div`
+  padding: 24px;
```

Comment:
I decided to punt on using `rem` for now, despite my earlier mentions of doing so in Slack, since the root `html` font size has been modified from its default `16px`, something that isn't recommended:

https://github.com/votingworks/arlo/blob/84b5494432ce94bc015c71ba0020c0bde7a72f99/client/src/App.css#L53-L56

I'd first want to remove that modification, update the few existing `rem` styles, and then start using `rem`

Reply:
Issue created ✅ https://github.com/votingworks/arlo/issues/1594